linode/manager
# refactor: [M3-6449] - React Query for Profile - Trusted Devices

## Description 📝

- Modernizes and React Query-ifys Trusted Devices visible on `http://localhost:3000/profile/auth`
- Removed the `ToggleState` component because it seems like an unnecessary layer of abstraction 

### Added Functionality ✨
- The ability to order the table by `Last IP`, `Last Used`, and `Expires`

## Preview 📷

![Screenshot 2023-03-30 at 12 12 05 AM](https://user-images.githubusercontent.com/115251059/228727278-81b6fa5f-9bc7-4d65-921f-1433ef985b89.jpg)

## How to test 🧪

- Trust a devices on your Linode account by checking `Trust this device for 30 days` when you login
- Go to `http://localhost:3000/profile/auth` and test the table found at the bottom of the page
- Also check functionality of the 2FA related components because the `ToggleState` component is no longer used
